matchbox_socket/src/webrtc_socket/error.rs

```diff
@@ -24,3 +25,46 @@ pub enum SignallingError {
     #[error("socket failure communicating with signalling server")]
     Socket(#[from] ws_stream_wasm::WsErr),
 }
+
+/// An error that can occur with WebRTC messaging.
+#[derive(Debug, thiserror::Error)]
+pub enum MessagingError {
```

Comment:
```suggestion
pub(crate) enum MessagingError {
```

Nit: If we're only have a single variant, we could use a single `SendError` tuple struct instead.

Reply:
Yeah, can always make it an enum again if needs be